Formula/calicoctl.rb

```diff
@@ -0,0 +1,16 @@
+class Calicoctl < Formula
+  desc "Calico CLI tool"
+  homepage "https://www.projectcalico.org"
+  url "https://github.com/projectcalico/calicoctl/releases/download/v1.6.1/calicoctl-darwin-amd64"
```

Comment:
This will need to be built from source.

Reply:
Is it preferable to build from source, instead of using the binary? Building from source takes unnecessary time. The binary works fine on any modern `x64` mac.

Reply:
Not only preferable but absolutely mandatory. 99.9% from 10.10-10.13 won't even see the from-source compile stage, so.

Reply:
Got it. I am not super familiar with the `go` building tools. Let me see if I can make the changes

Reply:
Looks like they're using `glide`. Check out something like [peco](https://github.com/Homebrew/homebrew-core/blob/3ed7f671da6676d3d30126975ccf59cef95aeb4e/Formula/peco.rb#L16-L28) as an example.